tests/src/test/java/com/cloudera/director/google/GoogleLauncherTest.java

```diff
@@ -59,6 +61,8 @@
 
   @BeforeClass
   public static void beforeClass() throws IOException {
+    Assume.assumeFalse(System.getProperty("GCP_PROJECT_ID", "").isEmpty());
```

Comment:
Nope because TestFixture.newTestFixture(false) would throw an exception. A bit more refactoring is needed to enable live tests. We should look into using encrypted environment variables https://docs.travis-ci.com/user/environment-variables/